docs/sources/reference/api/docker_remote_api.md

```diff
@@ -52,6 +52,21 @@ You can still call an old version of the API using
 You can now supply a `stream` bool to get only one set of stats and
 disconnect
 
+`GET /images/json`
+
+**New!**
+The `Created` field will be responsed as ISO 8601 String instead of UNIX epoch
```

Comment:
Not sure; do we already use/mention one of them in the docs? I did a quick search and according to http://stackoverflow.com/questions/522251/whats-the-difference-between-iso-8601-and-rfc-3339-date-formats, RFC3339 is a more strict in the way the date must be formatted. If our implementation is indeed strict RFC3339 (please check), that may be an important difference.